doc/man7/openssl-glossary.pod

```diff
@@ -66,6 +160,31 @@ This is specified in RFC 5208:
 
 L<https://tools.ietf.org/html/rfc5208>
 
+=item Property
+
+A property is a way of classifying and selecting algorithm implementations.
+A property is a key/value pair expressed as a string. For example all algorithm
+implementations in the default provider have the property "provider=default".
+An algorithm implementation can have multiple properties defined against it.
+
```

Comment:
Maybe mention it is used for fetching here or link back to the fetching item.

Reply:
I've added a reference to the Property Query Strings section here. I've added a sentence about fetching in Property Query Strings.